inline-caching/README.md

```diff
@@ -0,0 +1,255 @@
+
+By using GemFire's inline cache, frequently accessed data can be stored in GemFire's memory, reducing the number of queries that need to be made 
+to the database. This can help alleviate some of the scaling limits of the database, as it reduces the amount of load placed on the database server. 
+It can also help reduce network latency and improve overall system performance, as data can be retrieved locally rather than over a network connection.
+
+# Implementation
+Setting up an inline cache with GemFire and a database involves several steps. Here's an overview of the process.
+
+## Install and Configure GemFire
+
+The first step is to install and configure GemFire on your system.
+Download and install VMware GemFire from [Tanzu Network](https://network.pivotal.io). Follow the installation instructions in the [GemFire documentation](https://docs.vmware.com/en/VMware-GemFire/index.html).
+
+## Clone the GemFire examples repository from GitHub.
+
+There is a working code examples for how to setup an inline cache here.
+```text
+$ git clone git@github.com:gemfire/gemfire-examples.git
+```
+
+## Configure Access to GemFire Maven Repository
+
+The quickstart tutorial requires access to the VMware Commercial Maven Repository for the GemFire product jars. Please sign-up for access to repo at <https://commercial-repo.pivotal.io/register>.
+
+Create a `gradle.properties` file in this directory with the following information:
+
+```plain
+gemfireRepoUsername=<your commercial maven repository username>
+gemfireRepoPassword=<your commercial maven repository password>
+```
+
+These properties will be used in the `web` and `cache-loader` gradle projects. 
+
+We can now build our example with 
+```bash
+./gradlew build
+```
+
+## Database setup
+
+### Install Postgres
+You can use any database you want that has a JDBC driver for it. For this example, we will be using Postgres. 
+You can either download postgres from https://www.postgresql.org/download/ and install it manually or if on a Mac, use brew install.
+
+```bash
+brew install postgresql
+brew services start postgresql
+```
+
+### Configure Postgres
+
+Now that postgres is running, we will want to create some tables and a user. One of the easier ways to do this
+is to use the test database that postgres provides for benchmarking via `pgbench`.
+
+```bash
+pgbench -i -s 50 postgres
+```
+
+This will create a few tables for us that we can manipulate. We will also need a user that our GemFire client can use to access the database
+
+```bash
+psql postgres
+create user myuser with encrypted password 'mypass';
+grant all privileges on database postgres to myuser;
+grant all privileges on all tables in schema public to myuser;
+```
+
+### Download Postgres JDBC driver
+
+This will be necessary for GemFire to communicate to Postgres in our example
+
+```bash
+curl https://jdbc.postgresql.org/download/postgresql-42.6.0.jar > postgresql-42.6.0.jar
+```
```

Comment:
Can we not just add the following maven import into the gradle file
```implementation 'org.postgresql:postgresql:42.6.0'```

Reply:
cool idea. and it apparently works :)